src/script/cpp_api/s_security.cpp

```diff
@@ -109,7 +109,12 @@ void ScriptApiSecurity::initializeSecurity()
 		"string",
 		"table",
 		"math",
-		"bit"
+		"bit",
+		// Not sure if completely safe. But if someone enables tracy, they'll
+		// know what they do.
+#if BUILD_WITH_TRACY
+		"tracy",
+#endif
```

Comment:
Might we want a short notice in `lua_api.md` as well? e.g. link to other documentation.

Reply:
It's not part of our guaranteed lua api. It's a development tool.
For what purpose would we add it to `lua_api.md`? To advertise it to modders? Hmm, maybe we want that.

Reply:
Added something. Is this good?

Reply:
Yes (reference to the comments below)